internal/config/config.go

```diff
@@ -64,7 +79,19 @@ func parseConfig(cm *corev1.ConfigMap) error {
 
 	// Default values
 	var cfg = SkiperatorConfig{
-		TopologyKeys: []string{"kubernetes.io/hostname"},
+		TopologyKeys:                []string{"kubernetes.io/hostname"},
+		LeaderElection:              false,
+		LeaderElectionNamespace:     "skiperator-system",
```

Comment:
```suggestion
		LeaderElectionNamespace:     cmNamespace,
```

Should use constant, and that constant should probably be renamed to `skiperatorSystemNs` or something like that.